package.json

```diff
@@ -12,7 +12,7 @@
         "@mui/icons-material": "^5.5.1",
```

Comment:
There are dependencies to clean . Mainly those ones : 
"cheap-ruler": "^3.0.0",
                "deck.gl": "^8.9.18",
                "geolib": "^3.3.4",
                "mapbox-gl": "^3.0.0",
                "maplibre-gl": "^3.0.0",
                "react-map-gl": "^7.0.0"